.github/workflows/starknet-go-tests.yml

```diff
@@ -0,0 +1,33 @@
+name: starknet-go tests
+
+on:
+  workflow_call:
+    secrets:
+      TEST_RPC_URL:
+        required: false
+      TEST_ACCOUNT_PRIVATE_KEY:
+        required: false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v4
+        with:
+          repository: NethermindEth/starknet.go
+          ref: main
```

Comment:
nit: this could be an argument to this workflow, defaulting to main, so you can test releases there without changing much stuff around :)

Reply:
thanks, argument added